internal/infrastructure/kubernetes/infra.go

```diff
@@ -35,6 +35,7 @@ func NewInfra(cli client.Client) *Infra {
 	return &Infra{
 		mu:        sync.Mutex{},
 		Client:    cli,
+		Namespace: "envoy-gateway-system",
```

Comment:
isnt this being set here https://github.com/envoyproxy/gateway/blob/2ba39476943984774a31715e7b37275b58a50534/internal/infrastructure/manager.go#L33

Reply:
If the OS env is set to `ENVOY_GATEWAY_NAMESPACE`. I did not set it and we should use the default. Then the OS env can be used to override the default "envoy-gateway-system".

Reply:
sg, can we consolidate both the logic in one place using this PR

Reply:
Done in https://github.com/envoyproxy/gateway/commit/80c28911a654cf93c8959eb1c128312be6174292